### ISSUE TITLE
NullPointerException from JDA

### ISSUE THREAD

**Issue by ghost** [CraftTogether/chat-bridge#12]
```
[11:49:48 ERROR]: [net.dv8tion.jda.api.requests.RestAction] Encountered error while processing success consumer
java.lang.NullPointerException: Cannot invoke "java.awt.Color.getRed()" because "colour" is null
        at com.github.crafttogether.chatbridge.minecraft.MinecraftJoinEvent.lambda$assignColor$0(MinecraftJoinEvent.java:56) ~[chat-bridge-1.1-SNAPSHOT-3.jar:?]
        at net.dv8tion.jda.api.requests.Request.lambda$onSuccess$0(Request.java:95) ~[Kelp-1.0.jar:?]
        at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

**Comment by m5rian** [comment]
Should be fixed with b5435dad4e9b242436d6480dea7adda0e300a837

**Comment by m5rian** [comment]
Closing for now
